src/Symfony/Component/Validator/Constraints/Count.php

```diff
@@ -55,6 +58,7 @@ public function __construct(
         string $minMessage = null,
         string $maxMessage = null,
         string $divisibleByMessage = null,
+        string $conditionExpression = null,
```

Comment:
This is a Bc break, it needs to be added as last argument

Reply:
fixed